src/components/Layout/Header.js

```diff
@@ -0,0 +1,21 @@
+import { Fragment } from "react";
+import { Link } from "react-router-dom";
+
+import classes from "./Header.module.css";
+
+function Header() {
+  return (
+    <div>
+      <ul className={classes["nav-ul"]}>
+        <li>
+          <Link to="/timer">Timer</Link>
+        </li>
+        <li>
+          <Link to="/chat">Chat</Link>
+        </li>
+      </ul>
+    </div>
+  );
+}

```

Comment:
```suggestion
const navLinks = [
  {
    name: "Timer",
    path: "/timer"
  }....
]
function Header() {
  return (
    <div>
      <ul className={classes["nav-ul"]}>
        {navlinks.map(({ name, path }) => 
          <li>
            <Link to={path}>{name}</Link>
          </li>
        )
      </ul>
    </div>
  );
}

```

De esta manera cuando tengas que editar los links o lo que sea simplemente los arreglas al array